src/main/scala/junctions/MultiWidthFifo.scala

```diff
@@ -83,12 +83,12 @@ class MultiWidthFifoTest extends UnitTest {
   val big2little = Module(new MultiWidthFifo(16, 8, 8))
   val little2big = Module(new MultiWidthFifo(8, 16, 4))
 
-  val bl_send = Reg(init = Bool(false))
-  val lb_send = Reg(init = Bool(false))
-  val bl_recv = Reg(init = Bool(false))
-  val lb_recv = Reg(init = Bool(false))
-  val bl_finished = Reg(init = Bool(false))
-  val lb_finished = Reg(init = Bool(false))
+  val bl_send = Reg(init = false.B)
```

Comment:
Why not just go all the way to `RegInit(false.B)`? `Reg(init=)` is deprecated now.

Reply:
Because i'd rather not touch junctions._